package.json

```diff
@@ -31,7 +31,7 @@
     "eslint-config-next": "13.4.8",
     "kbar": "^0.1.0-beta.45",
     "lande": "^1.0.10",
-    "next": "14.0.1",
+    "next": "14.2.5",
```

Comment:
In latest version, they fixed an issue that would cause rsc requests to fail when `useSearchParams` is used in a component that is rendered in the page. we never had this issue until we had to pass useSearchParams at the top level.